lib/src/builder/codegen/component_factory_generator.dart

```diff
@@ -73,45 +73,34 @@ class ComponentFactoryProxyGenerator extends BoilerplateDeclarationGenerator {
           span: getSpan(sourceFile, parentType)));
     }
 
-    if (isComponent2) {
```

Comment:
Besides `builderFactory` changes, this is the same except DRY'd up

See this same diff [ignoring whitespace](https://github.com/Workiva/over_react/pull/484/files?w=1#diff-6894948510cfc7ab14bfa56b43b201fbL76) for a better reviewing experience